src/translations/main_zh_cn.ts

```diff
@@ -510,47 +510,47 @@ p, li { white-space: pre-wrap; }
-        <translation>支持非NETWM兼容的窗口管理器</translation>
+        <translation>支持非 NETWM 兼容的窗口管理器</translation>
     </message>
     <message>
         <source>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;If your desktop manager is not fully NETWM compliant, you may need to check this checkbox so it can detect Thunderbird window and is able to minimise and hide it.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</source>
-        <translation>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;如果您的桌面环境不是NETWM兼容的，您可能需要勾选此项我们才能正确探测到Thunderbird，并能够将其最小化和隐藏。&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</translation>
+        <translation>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;如果您的桌面环境不是 NETWM 兼容的，您可能需要勾选此项才能允许桌面环境正确探测 Thunderbird 并将其最小化和隐藏。&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</translation>
     </message>
     <message>
         <source>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;If enabled, this option adds the &amp;quot;Ignore currently unread emails&amp;quot; action to the context menu. This action allows you to ignore the emails which are currently unread. Birdtray would then pretend there are no unread emails left, and would only show new emails over the ignored count.&lt;/p&gt;&lt;p&gt;&lt;br/&gt;For example, if there were 10 unread emails, and you clicked on &amp;quot;Ignore&amp;quot; action, Birdtray will show no unread email indicator as long as unread email count remains at 10. Once new email is received and you have 11 total unread emails, Birdtray will show the new email count as 1.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</source>
-        <translation type="unfinished"></translation>
+        <translation>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;勾选此项会将&amp;quot;忽略当前未读邮件&amp;quot;添加到右键菜单。该动作允许您忽略当前的未读邮件；Birdtray 会伪装成已经没有未读邮件，且只将此后的新邮件纳入未读计数。&lt;/p&gt;&lt;p&gt;&lt;br/&gt;例如，如果您在有 10 封未读邮件时执行&amp;quot;忽略&amp;quot;动作，只要未读邮件数仍为 10 封，Birdtray 就会在计数器显示没有未读邮件。此后如果收到一封新邮件、总未读邮件数变为 11，Birdtray 将在计数器显示有 1 封未读邮件。&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</translation>
     </message>
     <message>
         <source>At startup</source>
-        <translation type="unfinished"></translation>
+        <translation>启动时</translation>
     </message>
     <message>
         <source>When hiding Thunderbird</source>
-        <translation type="unfinished"></translation>
+        <translation>隐藏 Thunderbird 时</translation>
     </message>
     <message>
         <source>When showing Thunderbird</source>
-        <translation type="unfinished"></translation>
+        <translation>显示 Thunderbird 时</translation>
     </message>
     <message>
         <source>Start Thunderbird by clicking on the tray icon if it was closed</source>
-        <translation type="unfinished"></translation>
+        <translation>如果 Thunderbird 已关闭，在点击托盘图标时启动 Thunderbird</translation>
     </message>
     <message>
         <source>When detecting a change, wait</source>
-        <translation type="unfinished"></translation>
+        <translation>检测到改变时，等待</translation>
```

Comment:
personally, "变更" is better. but it's up to you~
```suggestion
        <translation>检测到变更时，等待</translation>
```